notebooks/04.02-Simple-Scatter-Plots.ipynb

```diff
@@ -37,10 +37,8 @@
   },
```

Comment:
Some nice colorful output. Now that's what I call a scatter plot! <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=19>ReviewNB</a>_